src/test/scala/dataclass/ZeroFieldTests.scala

```diff
@@ -44,28 +44,32 @@ object ZeroFieldTests extends TestSuite {
         @data class Bar[T]()
         val barI = Bar[Int]()
         val barS = Bar[String]()
-        assert(barI == barS) // type erasure, both instances are the same at runtime
+        // type erasure, both instances are the same at runtime
```

Comment:
I made this change because `maxColumn=80` and it would result in weird format. increasing `maxColumn` would have caused more changes.